docs/community.md

```diff
@@ -1,6 +1,5 @@
+# How to get in contact with the team
 
-How to get in contact with the team.
-
-* [TrenchBoot Google Group](https://groups.google.com/g/trenchboot-devel)
-* [Twitter](https://twitter.com/TrenchBoot)
-* [#trenchboot channel on OSFW Slack](https://slack.osfw.dev)
+- [TrenchBoot Google Group](https://groups.google.com/g/trenchboot-devel)
+- [Twitter](https://twitter.com/TrenchBoot)
+- [#trenchboot channel on OSFW Slack](https://slack.osfw.dev)
```

Comment:
```suggestion
- [TrenchBoot room on Matrix](https://matrix.to/#/#OSFW-Trenchboot:matrix.org)
```

Slack channel is pretty much dead.

Reply:
fixed 38411e46ed59e835bd55a8351bc3755265a420b3